src/EventSubscriber/RequestSubscriber.php

```diff
@@ -248,7 +249,7 @@ protected function constructQuerySpan($connection, array $query) {
         'abs_path' => $query['caller']['file'],
         'filename' => substr($query['caller']['file'], strrpos($query['caller']['file'], '/') + 1),
         'lineno' => $query['caller']['line'],
-        'vars' => $query['args'],
+        'vars' => (object) $query['args'],
```

Comment:
There must be queries where args is a numerically indexed array because the APM was giving a 400 error - `vars` was an array instead of an object or null. I'm not certain if there are any other parts of a span we might incorrectly encode as an array.